i6_models/parts/frontend/common.py

```diff
@@ -0,0 +1,42 @@
+from typing import Tuple, Union
+
+import torch
+from torch import nn
+
+
+IntTupleIntType = Union[int, Tuple[int, int]]
+
+
+def _get_padding(input_size: Union[int, Tuple[int, ...]]) -> Union[int, Tuple[int, ...]]:
```

Comment:
I don't think it should be private here.
```suggestion
def get_padding(input_size: Union[int, Tuple[int, ...]]) -> Union[int, Tuple[int, ...]]:
```
But actually, I would try to make the name more expressive.
```suggestion
def get_same_padding(input_size: Union[int, Tuple[int, ...]]) -> Union[int, Tuple[int, ...]]:
```